app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -1,3 +1,5 @@
+require 'hawkular/hawkular_client_utils'
```

Comment:
same here. Maybe this require should be somewhere else, like in the connect method in the manager class?